xr_sample_project/Assets/SampleViewer/Samples/XRTableTop/readme.md

```diff
@@ -0,0 +1,67 @@
+# Explore the World In Table Top VR
+
+Move around in an ArcGIS World using Virtual Reality to move and fly through different cities
+![Image of XR Table Top](XRTableTop.png)
+
+## How to use the sample
+
+There are two ways to use this headset, Desktop VR and Android VR. This sample utilizes the OpenXR framework and supports the following headsets/devices for **Desktop VR**:
+- Meta Rift
+- Meta Rift S
+- Meta Quest
+- Meta Quest 2
+- Meta Quest Pro
+- Meta Quest 3
+- HTC Vive
+- HTC Vive Pro
+- HTC Vive Cosmos
+- Valve Index
+
+This sample utilizes the OpenXR framework and supports the following headsets/devices for **Android VR**:
+- Meta Quest
+- Meta Quest 2
+- Meta Quest Pro
+- Meta Quest 3
+- HTC Vive XR Elite
+
+**For Desktop Use**
+1. Ensure your headset is properly connected to your computer. For Meta Headsets, make sure the Meta Desktop App is open and your headset says **connected** in the **devices** section. For Steam VR headsets, ensure Steam is open and Steam VR is running. 
+2. Place the headset on your head. The sample should automatically use the headset to control the camera's position and rotation in the world. 
+3. To move, use the joystick on the left controller, pushing the joystick (on a Quest controller) or moving on the touchpad (on a Vive controller) will move you in that direction. Move up and down by pushing the right joystick (on a Quest controller) forward and backward, or moving forward and backward on the touchpad (on a Vive controller).The right controller can also be used to 'snap turn' 45 degrees, making it so you can turn around quickly. Select the snap turn option, push the joystick left and right (on a Quest controller) or swipe left and right (on a Vive controllers) to snap turn. 
+4. Use the menu button on controllers to open/close the menu, pull the left or right trigger to interact with the menu. 
+
+**Note: AR Foundation is installed and is used to allow Passthrough on the Meta Quest Devices. If you would like to have Passthrough on other devices, please refer to the developer section for that headset**
+**For Android Use**
```

Comment:
That's a good idea depending on how many changes are necessary (does the UI need to be changed then too?).  I'm fine with either option (conditional macros or doc notes). But even if we take the conditional path, we might need to mention something here because it would seem strange if some of the presets are missing after the build.

Reply:
UI can stay the same, just when you cycle to Redlands or Christchruch there won't be a BSL there. But both of those locations have other meshes there so it's not like it would just be terrain.